test/templates/template.yaml.example.311.43

```diff
@@ -0,0 +1,50 @@
+genevaLoggingAccount: ccpopenshiftdiag
+genevaLoggingControlPlaneAccount: RPOpenShiftAccount
+genevaLoggingNamespace: CCPOpenShift
+genevaLoggingSector: US-Test
+genevaMetricsAccount: RPOpenShift
+genevaMetricsEndpoint: https://az-int.metrics.nsatc.net/
+imageOffer: osa
+imagePublisher: redhat
+imageSku: osa_311
+imageVersion: 311.43.20181121
+images:
+  alertManagerBase: registry.access.redhat.com/openshift3/prometheus-alertmanager
+  ansibleServiceBroker: registry.access.redhat.com/openshift3/ose-ansible-service-broker:v3.11.43
+  azureControllers: quay.io/openshift-on-azure/azure-controllers:v3.11
+  clusterMonitoringOperator: registry.access.redhat.com/openshift3/ose-cluster-monitoring-operator:v3.11.43
+  configReloaderBase: registry.access.redhat.com/openshift3/ose-configmap-reloader
+  console: registry.access.redhat.com/openshift3/ose-console:v3.11.43
+  controlPlane: registry.access.redhat.com/openshift3/ose-control-plane:v3.11.43
+  etcdBackup: quay.io/openshift-on-azure/etcdbackup:latest
+  format: registry.access.redhat.com/openshift3/ose-${component}:${version}
+  genevaLogging: osarpint.azurecr.io/acs/mdsd:12051806
+  genevaStatsd: osarpint.azurecr.io/acs/mdm:git-a909a2e76
+  genevaTDAgent: osarpint.azurecr.io/acs/td-agent:latest
+  grafanaBase: registry.access.redhat.com/openshift3/grafana
+  kubeRbacProxyBase: registry.access.redhat.com/openshift3/ose-kube-rbac-proxy
+  kubeStateMetricsBase: registry.access.redhat.com/openshift3/ose-kube-state-metrics
+  genevaImagePullSecret: e2F1dGhzOntvc2FycGludC5henVyZWNyLmlvOnt1c2VybmFtZTp1c2VybmFtZSxwYXNzd29yZDpub3R0aGVwYXNzd29yZHlvdWFyZWxvb2tpbmdmb3IsZW1haWw6b3BlbnNoaWZ0QG1pY3Jvc29mdC5jb20sYXV0aDpkWE5sY201aGJXVTZibTkwZEdobGNHRnpjM2R2Y21SNWIzVmhjbVZzYjI5cmFXNW5abTl5Q2c9PX19fQo=
+  masterEtcd: registry.access.redhat.com/rhel7/etcd:3.2.22
+  metricsBridge: quay.io/openshift-on-azure/metricsbridge:latest
+  node: registry.access.redhat.com/openshift3/ose-node:v3.11.43
+  nodeExporterBase: registry.access.redhat.com/openshift3/prometheus-node-exporter
+  oAuthProxyBase: registry.access.redhat.com/openshift3/oauth-proxy
+  prometheusBase: registry.access.redhat.com/openshift3/prometheus
+  prometheusConfigReloaderBase: registry.access.redhat.com/openshift3/ose-prometheus-config-reloader
+  prometheusOperatorBase: registry.access.redhat.com/openshift3/ose-prometheus-operator
+  registry: registry.access.redhat.com/openshift3/ose-docker-registry:v3.11.43
+  registryConsole: registry.access.redhat.com/openshift3/registry-console:v3.11.43
+  router: registry.access.redhat.com/openshift3/ose-haproxy-router:v3.11.43
+  serviceCatalog: registry.access.redhat.com/openshift3/ose-service-catalog:v3.11.43
+  sync: quay.io/openshift-on-azure/sync:latest
+  templateServiceBroker: registry.access.redhat.com/openshift3/ose-template-service-broker:v3.11.43
+  webConsole: registry.access.redhat.com/openshift3/ose-web-console:v3.11.43
+runningUnderTest: true
```

Comment:
With @asalkeld's PR for removing the locally created users and moving to a aad group sync I believe this can be removed.  In order to move this along you might want to do in a follow up.